fzf-marks.plugin.bash

```diff
@@ -22,12 +22,25 @@
 
 command -v fzf >/dev/null 2>&1 || return
 
+PROGRAM_NAME="${PROGRAM_NAME:-fzf-marks}"
+
+CONFIG_DIR="${XDG_CONFIG_HOME:-$HOME/.config}/$PROGRAM_NAME" #currently unused, but in the future can be used to load (and store?) an .env file from to more easily switch between multiple configuration
+DATA_DIR="${XDG_DATA_HOME:-$HOME/.local/share}/$PROGRAM_NAME"
+LEGACY_FZF_MARKS_FILE="${HOME}/.fzf-marks"
+
 if [[ -z ${FZF_MARKS_FILE-} ]] ; then
-    FZF_MARKS_FILE=${HOME}/.fzf-marks
+    FZF_MARKS_FILE="${DATA_DIR}/default_bookmarks.fzf-marks"
 fi
 
-if [[ ! -f ${FZF_MARKS_FILE} ]]; then
+if [[ ! -f "${FZF_MARKS_FILE}" ]]; then
+  # Fallback support for the previous default location of the bookmarks file.
+  if [[ -f "$LEGACY_FZF_MARKS_FILE" ]]; then
+    echo "Your FZF_MARKS_FILE is still in its legacy location of $LEGACY_FZF_MARKS_FILE; you might want to consider moving it to "$FZF_MARKS_FILE" so it follows the XDG standard, or explicitly set FZF_MARKS_FILE='$LEGACY_FZF_MARKS_FILE'." >&2
```

Comment:
Are the quotes around "$FZF_MARKS_FILE" necessary here? If not, it might be good to remove them, to be more consistent ­— indeed, there are no quotes around $LEGACY_FZF_MARKS_FILE.

Reply:
I believe I already changed that based on @akinomyoga 's suggestions :)
See 462bfd9 :)

Reply:
Looks good, many thanks! :)